posthog/hogql_queries/insights/trends/trends_query_runner.py

```diff
@@ -384,14 +384,18 @@ def get_value(name: str, val: Any):
             if series.aggregate_values:
                 series_object = {
                     "data": [],
-                    "days": [
-                        item.strftime(
-                            "%Y-%m-%d{}".format(" %H:%M:%S" if self.query_date_range.interval_name == "hour" else "")
-                        )
-                        for item in get_value("date", val)
-                    ]
-                    if response.columns and "date" in response.columns
-                    else [],
+                    "days": (
+                        [
+                            item.strftime(
+                                "%Y-%m-%d{}".format(
+                                    " %H:%M:%S" if self.query_date_range.interval_name == "hour" else ""
+                                )
```

Comment:
Nit: I'd say concatenation with `+` would be more readable than `.format()`

Reply:
Is this used as label? There is a function for that used by other runners.

Reply:
There is `format_label_date`, but that format hourly as `%-d-%b-%Y %H:%M` instead of `%Y-%m-%d %H:%M:%S`. Don't know if this would break something, so opting to do this in a separate PR.